src/components/autocomplete/autocomplete-select-interface.ts

```diff
@@ -0,0 +1,20 @@
+/* eslint-disable @typescript-eslint/no-explicit-any */
+export interface AutocompleteOption {
+  // eslint-disable-next-line @typescript-eslint/no-explicit-any
+  value: any;
+  // eslint-disable-next-line @typescript-eslint/no-explicit-any
+  label: any;
+  // eslint-disable-next-line @typescript-eslint/no-explicit-any
+  bulkOption?: any;
+  // eslint-disable-next-line @typescript-eslint/no-explicit-any
+  status?: any;
+}
+
+export interface AutocompleteChangeEventDetail {
+  value: any | any[] | undefined | null;
```

Comment:
it should have a type (avoid use of any)
I think the value will always be string